common/src/test/java/org/apache/sedona/common/raster/GeometryFunctionsTest.java

```diff
@@ -76,6 +77,49 @@ public void testConvexHullFromRasterFile() throws FactoryException, TransformExc
         assertEquals(expectedCoordOne.getY(), coordinates[4].getY(), 0.5d);
     }
 
+    @Test
+    public void testMinConvexHull() throws FactoryException, TransformException {
```

Comment:
Do these tests yield the same results as PostGIS?

Reply:
Yes, all tests are done both on sedona and PostGIS